app/views/layouts/_navbar.html.erb

```diff
@@ -16,23 +16,22 @@
     </button>
     <div class="collapse navbar-collapse" id="navbarButtonsExample">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-        <li class="nav-item">
-          <%= link_to 'Home', root_path, class: "nav-link active" %>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="#">Orders</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="#">Pay</a>
-        </li>
+        <% if cookies[:auth_token] %>
+          <li class="nav-item">
+            <%= link_to 'Home', root_path, class: "nav-link active" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to 'Ingredients', ingredients_path, class: "nav-link" %>
```

Comment:
Add class `active` to the current heading.
If we are on `Ingredients` page, then `Ingredients` should be active